pkg/jobrunaggregator/jobruntablecreator/table_creator.go

```diff
@@ -0,0 +1,37 @@
+package jobruntablecreator
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"cloud.google.com/go/bigquery"
+
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorapi"
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorlib"
+)
+
+type allJobRunTableCreatorOptions struct {
+	ciDataClient jobrunaggregatorlib.CIDataClient
+	ciDataSet    *bigquery.Dataset
+}
+
+func (r *allJobRunTableCreatorOptions) Run(ctx context.Context) error {
+
+	// Create JobRunTable
+	jobRunTable := r.ciDataSet.Table(jobrunaggregatorlib.JobsTableName)
+	_, err := jobRunTable.Metadata(ctx)
+	if err != nil {
+		schema, err := bigquery.InferSchema(jobrunaggregatorapi.JobRow{})
+		if err != nil {
+			return err
+		}
+		if err := jobRunTable.Create(ctx, &bigquery.TableMetadata{Schema: schema}); err != nil {
+			return err
+		}
+	} else {
+		fmt.Fprintf(os.Stderr, "table already exists: %s\n", jobrunaggregatorlib.JobRunTableName)
```

Comment:
I feel like this should not be an Stderr as virtually all executions would print this line. If that was in Stephen's module as well it should probably be changed as well, as this is not really an error just expected behavior.